doc/client/play29.md

```diff
@@ -0,0 +1,71 @@
+# Using as a Play 2.9 client
+
+Add the dependency:
+
+```scala
+"com.softwaremill.sttp.tapir" %% "tapir-play-client" % "@VERSION@"
```

Comment:
the artifact name here is wrong - I think it would really be better to combine the docs into a single page, just with mentioning alternative artifact names (plus one or two akka/pekko imports) - these are all the differences